src/app/api/bridge/user/new/get-status/route.ts

```diff
@@ -1,13 +1,34 @@
-import type { NextRequest } from 'next/server'
-import { NextResponse } from 'next/server'
+import { NextResponse, NextRequest } from 'next/server'
 import * as interfaces from '@/interfaces'
+import { GET as getUserFromCookie } from '@/app/api/peanut/user/get-user-from-cookie/route'
 
 export async function POST(request: NextRequest) {
     try {
         const { userId, type } = await request.json()
         if (!process.env.BRIDGE_API_KEY) {
             throw new Error('BRIDGE_API_KEY is not defined')
         }
+
+        const getUserFromCookieRequest = new NextRequest(
+            `${process.env.NEXT_PUBLIC_BASE_URL}/api/peanut/user/get-user-from-cookie`,
+            {
+                method: 'GET',
+                headers: {
+                    cookie: request.headers.get('cookie') ?? '',
+                    ...request.headers,
+                },
+            }
+        )
+        const getUserFromCookieResponse = await getUserFromCookie(getUserFromCookieRequest)
```

Comment:
what's going on here, why

Reply:
We need the user data, of the calling user, so we need to ensure that the user is authenticated and get its data. We can't trust on input